common/flatpak-dir.c

```diff
@@ -3908,20 +3908,20 @@ repo_pull (OstreeRepo                           *self,
 {
   gboolean force_disable_deltas = (flatpak_flags & FLATPAK_PULL_FLAGS_NO_STATIC_DELTAS) != 0;
   g_autofree char *current_checksum = NULL;
+  g_autofree char *owned_rev_to_fetch = g_strdup (rev_to_fetch);
   g_autoptr(GVariant) old_commit = NULL;
   g_autoptr(GVariant) new_commit = NULL;
   const char *revs_to_fetch[2];
   gboolean res = FALSE;
   g_autofree gchar *collection_id = NULL;
   g_autoptr(GError) dummy_error = NULL;
+  gboolean pulled_using_p2p = FALSE;
+  OstreeCollectionRef collection_ref;
 
   /* The ostree fetcher asserts if error is NULL */
   if (error == NULL)
     error = &dummy_error;
 
-  /* If @results_to_fetch is set, @rev_to_fetch must be. */
-  g_assert (results_to_fetch == NULL || rev_to_fetch != NULL);
```

Comment:
Maybe add a `/* (nullable) */` annotation to the `rev_to_fetch` argument now. I think that would be correct?

Reply:
I think those annotations aren't used much for internal functions but sure

Reply:
It’s more for the benefit of anyone reading the code. Thanks.